playbook/app/pb_kits/playbook/pb_table/docs/_table_header.html.erb

```diff
@@ -1,58 +1,69 @@
-<%= pb_rails("table", props: { size: "lg"} ) do %>
+<% data_rows = [
+    { 'territory' => 'North', 'firstname' => 'John', 'lastname' => 'Doe', 'age' => 30, 'job' => 'Engineer' },
+    { 'territory' => 'South', 'firstname' => 'Alice', 'lastname' => 'Smith', 'age' => 28, 'job' => 'Designer' },
+    { 'territory' => 'East', 'firstname' => 'Mike', 'lastname' => 'Johnson', 'age' => 35, 'job' => 'Manager' },
+    { 'territory' => 'West', 'firstname' => 'Sarah', 'lastname' => 'Brown', 'age' => 29, 'job' => 'Developer' },
+    { 'territory' => 'Central', 'firstname' => 'David', 'lastname' => 'Wilson', 'age' => 32, 'job' => 'Analyst' }
+  ]
+%>
+
+<% if params["sort"] %>
```

Comment:
I think it might be good to add a comment saying this is just for this example. I would expect this to be done in the users controller. Might say something like that. Ruby-ists woudl say this is bad code.

Reply:
We added a disclaimer to the doc md file under the doc example. Would you like me to add code comments as well?